db/seeds.rb

```diff
@@ -24,19 +24,8 @@
 	i.hc_prep = seed.rand(1000).to_s + quantity[seed.rand(4)]
 	i.bb_prep = seed.rand(1000).to_s + quantity[seed.rand(4)]
 	i.buckley_prep = seed.rand(1000).to_s + quantity[seed.rand(4)]
-	d.ingredients.append(i)
```

Comment:
You can leave this seeds file as it is. I am working on overwriting this entire file. 
It is fine right now, you don't need to revert your change.